tests/sentry/notifications/utils/test_participants.py

```diff
@@ -331,10 +361,177 @@ def test_fallthrough(self):
 
     def test_without_fallthrough(self):
         ProjectOwnership.objects.get(project_id=self.project.id).update(fallthrough=False)
-        event = self.store_event("no_rule.cpp")
+        event = self.store_event_owners("no_rule.cpp")
 
         assert self.get_send_to_owners(event) == {}
 
+    @with_feature("organizations:streamline-targeting-context")
+    def test_send_to_suspect_committers(self):
```

Comment:
Test `get_suspect_committers` where the `organizations:commit-context` feature flag isn't on (see [here](https://github.com/getsentry/sentry/blob/29c0a9ff0bdd1c79eed8e4e43a5a7558681c0c3a/src/sentry/utils/committers.py#L328))